tests/scalers/rabbitmq-helpers.ts

```diff
@@ -52,15 +55,15 @@ export class RabbitMQHelper {
 const publishYaml = `apiVersion: batch/v1
 kind: Job
 metadata:
-  name: rabbitmq-publish
+  name: rabbitmq-publish-{{QUEUE_NAME}}
 spec:
   template:
     spec:
       containers:
       - name: rabbitmq-client
-        image: jeffhollan/rabbitmq-client:dev
+        image: jorturfer/tests-rabbitmq
```

Comment:
Can we copy this image to a keda-owned account? If we're going to have something custom I don't think it should be from a person.

Reply:
it's in process https://github.com/kedacore/test-tools/pull/17 :)
I explained it in the PR description, basically I created the PR with WIP until the final image is ready

Reply:
The image is up: https://github.com/kedacore/test-tools/pkgs/container/tests-rabbitmq

Reply:
here we go!